h/static/scripts/annotator/adder.js

```diff
@@ -102,9 +102,12 @@ function Adder(container, options) {
     // property rather than `display` so that we can compute its size in order to
     // position it before display.
     display: 'block',
-    position: 'absolute',
     visibility: 'hidden',
 
+    // take position out of flow and off screen initially
+    position: 'absolute',
+    top: '100%',
```

Comment:
It doesn't matter, actually. The most important thing is that we set a value for top, bottom, left, or right so that it will _actually_ take on the absolutely positioned property. I can set it to zero though - might prevent confusion later on.